### DISCUSSION TITLE
What is the difference?

### DISCUSSION THREAD

**Discussion by Dhyfer1** [pear-devs/pear-desktop#1008] [General]
Hi @th-ch 

My O.S. is Windows and in the releases section I see two files that can help me: YouTube-Music-1.19.0.exe and YouTube-Music-Setup-1.19.0.exe. What is the difference between the two? which one should I install? Isn't there a portable version of youtube-music?

Cheers.

**Comment by Zo-Bro-23** [comment]
`YouTube-Music.exe` is a portable version, and `YouTube-Music-Setup.exe` is a setup version. Portable means that you run the exe directly to run the app. It doesn't install anything to your system, and it is standalone. The setup version will install the app to your system so that you can open it from the start menu, auto update the app, etc.

**Reply by Araxeus** [reply]
I'll add that you probably should install it unless you have a specific reason not to.

The standalone has some weird bugs sometimes
